### PR TITLE
Remove copy of old yubico library and replace it with surfnet library

### LINKED ISSUE #1: Replace library
History:
https://github.com/simplesamlphp/simplesamlphp/issues/36
https://code.google.com/archive/p/simplesamlphp/issues/479

The current external library in `libextinc/` is horribly dated and most importantly unmaintained by Yubico.. Yubico did release a 2.6 version recently, but it's still not namespaced, not composer-installable and issues/PR's for that are open for many years..
The best replacement seems to be https://github.com/enygma/yubikey

### PR DESCRIPTION
**Pull request by tvdijen**
Closes #1

### CODE REVIEW

**Comment by tvdijen** [comment]
For the failing PHP 8.0 tests, a PR was filed here; https://github.com/SURFnet/yubikey-api-client/pull/7

**Review by thijskh — APPROVED** [review]

**Comment by tvdijen** [comment]
PR upstream was merged; waiting for a tagged release

**Comment by tvdijen** [comment]
Release was tagged, only needs testing before merging

**Pull request merged**
